api/app.py

```diff
@@ -87,6 +127,7 @@ def run(url=None):
 
         socket.setdefaulttimeout(None)
 
+    initialize()
```

Comment:
The `run` function here isn't ever called in the production environment, its only used when running locally. So in production this initialize function will never get called.

Reply:
When I try running this in the dockerized environment I see this exception in the logs and a 502 error come back: 
```
  File "/var/www/circulation/env/lib/python3.8/site-packages/flask/app.py", line 2551, in __call__
    return self.wsgi_app(environ, start_response)
  File "/var/www/circulation/env/lib/python3.8/site-packages/flask/app.py", line 2531, in wsgi_app
    response = self.handle_exception(e)
  File "/var/www/circulation/env/lib/python3.8/site-packages/flask/app.py", line 1727, in handle_exception
    server_error = self.ensure_sync(handler)(server_error)
  File "/var/www/circulation/api/routes.py", line 98, in wrapped_function
    patron_web_domains = app.manager.patron_web_domains
AttributeError: 'Flask' object has no attribute 'manager'
```

Reply:
Okay - I'll fix that.